examples/server-worker/index.ts

```diff
@@ -51,30 +51,26 @@ app.post('/key-shares/generate', async (req: Request, res: Response) => {
       .json({ message: 'Keystore password is required' });
   }
 
-  const ssvKeys = new SSVKeys(SSVKeys.VERSION.V3);
-  const privateKey = await ssvKeys.getPrivateKeyFromKeystoreData(keystore, password)
-  const encryptedShares = await ssvKeys.buildShares(privateKey, operators_ids, operators_keys);
+  const ssvKeys = new SSVKeys();
```

Comment:
Even if you will init bls only once, you still need to extract `new SSVKeys` into level outside of express route